src/IB/IBFEInstrumentPanel.cpp

```diff
@@ -563,6 +559,10 @@ IBFEInstrumentPanel::initializeHierarchyIndependentData(IBFEMethod* ib_method_op
             d_U_dof_idx[jj].push_back(U_dof_index);
         }
     }
+    
+    // output the nodes of each meter mesh into a .dat file
+    outputNodes();
```

Comment:
If the point is to handle output outside of the class, should this call to `outputNodes()` still be in the implementation?